api/src/db/models.rs

```diff
@@ -0,0 +1,22 @@
+//! Contains objects that store data
+//!
+//! Data are stored in struct for database interaction
+//! such as retrieve or insert.
+
+use super::schema::users;
+
+// User store retrieved data from the user table
+#[derive(Queryable, Debug, Deserialize, Serialize)]
+pub struct User {
```

Comment:
https://rust-lang-nursery.github.io/api-guidelines/interoperability.html#c-common-traits

Reply:
done (Copy and Display not allowed)